LICENSE

```diff
@@ -199,287 +199,3 @@ Apache License
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
```

Comment:
Please note that in the link that you shared above (http://www.apache.org/dev/licensing-howto.html#bundle-asf-product), it says:

> Dependencies which are not included in the distribution MUST NOT be added to LICENSE and NOTICE. As far as LICENSE and NOTICE are concerned, only bundled bits matter

This means SINGA does not need any third party license because no third party "bits" are actually bundled with it.